pkg/pool-manager/pool_test.go

```diff
@@ -355,7 +355,7 @@ var _ = Describe("Pool", func() {
 			newVM.Name = "newVM"
```

Comment:
can you please update the terminology of the dryRun options?
```
In case of "server/All" dryRun, we do want to return an updated...
```

Reply:
updated PR desc